lighthouse-core/scripts/github-actions-commit-range.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
seems like we've started putting some scripts in `.github`, preference for moving this there?

Reply:
yea